src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -138,7 +138,7 @@ const useGroupInfo = () => {
         },
         (res) => {
           if (!res.didCancel) {
-            uploadImageBase64(res?.assets?.[0]?.base64);
+            uploadImage(res?.assets?.[0]?.uri);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The call to `uploadImage` now passes a URI instead of a base64 string. This change should be compatible with the updated `uploadImage` function, but ensure that the URI is correctly formed and accessible.



<!-- This is an auto-generated comment by OSS CodeRabbit -->